bin/existing-eks-mixed-observability.ts

```diff
@@ -1,4 +1,4 @@
-import ExistingEksMixedConstruct from '../lib/existing-eks-mixed-observability-construct';
+import ExistingEksMixedConstruct from '../lib/existing-eks-mixed-observability';
```

Comment:
Why is class name `ExistingEksMixedConstruct ` still having `Construct` ? We want to totally get rid of word Construct with `Pattern`. Also we dont want `-construct` to be removed. We would rather want it to be renamed to `-pattern`.